fhir-parent/pom.xml

```diff
@@ -164,7 +164,7 @@
             <dependency>
                 <groupId>jakarta.xml.ws</groupId>
                 <artifactId>jakarta.xml.ws-api</artifactId>
-                <version>2.3.3</version>
+                <version>3.0.1</version>
```

Comment:
not sure the implications.... it's only provided.

Reply:
the comment says its only used from fhir-server-test, but then i'm confused about why its marked provided